certora/helpers/MerkleTree.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+struct Leaf {
+    address addr;
+    address reward;
+    uint256 value;
+}
+
+struct InternalNode {
+    bytes32 id;
```

Comment:
We could, but then if there is a collision we need to change the contract. This might not be an issue in practice, but in general when designing certificates it's a good practice to leave room to fix steps that might fail